draft-ietf-quic-tls.md

```diff
@@ -1184,34 +1184,38 @@ least 3 bytes of frames in the unprotected payload if the packet number is
 encoded on a single byte, or 2 bytes of frames for a 2-byte packet number
 encoding.
 
-The sampled ciphertext for a packet with a short header can be determined by the
-following pseudocode:
+The sampled ciphertext can be determined by the following pseudocode:
 
 ~~~
-sample_offset = 1 + len(connection_id) + 4
+sample_offset = packet_number_offset + 4
```

Comment:
```suggestion
# pn_offset is the start of the Packet Number field.
sample_offset = pn_offset + 4
```

We already use `pn_offset` in the header protection section above.

Reply:
Thank you for the suggestion. Applied as part of fd01f88.